hack/scripts/payload/CreateMicroVM.json

```diff
@@ -29,8 +29,11 @@
     ],
     "interfaces": [
       {
-        "guest_device_name": "eth1",
-        "type": 0
+        "device_id": "eth1",
```

Comment:
Kind of spam, committed the file because:
1. it was easier for now
2. `guest_device_name` is `device_id`, but this payload wasn't updated.